packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -136,14 +136,20 @@ const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
         fetchOptions
     }
 
+    const baseUrl = config.proxy.split(MOBIFY_PATH)[0]
+    const privateClientEndpoint = `${baseUrl}${SLAS_PRIVATE_PROXY_PATH}`
+
     const apiClients = useMemo(() => {
         return {
             shopperBaskets: new ShopperBaskets(config),
             shopperContexts: new ShopperContexts(config),
             shopperCustomers: new ShopperCustomers(config),
             shopperExperience: new ShopperExperience(config),
             shopperGiftCertificates: new ShopperGiftCertificates(config),
-            shopperLogin: new ShopperLogin(config),
+            shopperLogin: new ShopperLogin({
+                ...config,
+                proxy: enablePWAKitPrivateClient ? privateClientEndpoint : config.proxy
+            }),
```

Comment:
We set up shopperLogin also in auth.js. I wonder if we should make it so that we only have one instance of shopperLogin rather than separate ones in auth.js and the provider?

Reply:
I was thinking the same thing. Unfortunately the `auth.js` file is just a standard javascript file and not a react hook where I would be able to access the existing ShopperLogin instance. Alternatively you can create a util to create that client for you, but I don't feel like there is much value in doing that at this point in time.